iothub/device/src/MessageAcknowledgementType.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using static System.Net.WebRequestMethods;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// The options for acknowledging a cloud to device message.
+    /// </summary>
+    /// <seealso href="https://learn.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-c2d#the-cloud-to-device-message-life-cycle"/>
+    public enum MessageAcknowledgementType
+    {
+        /// <summary>
+        /// The message will be positively acknowledged. This removes the message from the queue
+        /// and it will not be sent again.
+        /// </summary>
```

Comment:
Use remarks for extra info. Also below.

```suggestion
        /// <summary>
        /// The message will be positively acknowledged.
        /// </summary>
        /// <remarks>
        /// This removes the message from the queue; it will not be sent again.
        /// </remarks>
```